src/components/Main/Main.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem';
+
+type Props = {
+  todos: Todo[];
+};
+
+export const Main: React.FC<Props> = ({ todos }) => {
+  return (
+    <ul className="todoapp__main" data-cy="TodoList">
+      {todos.map(todo => {
+        const { title, completed, id } = todo;
+
+        return <TodoItem title={title} completed={completed} id={id} />;
+      })}
```

Comment:
also where is `key` prop?
```suggestion
      {todos.map(({ title, completed, id })=> ( <TodoItem title={title} completed={completed} id={id} />;
      ))}
```